experiments/Manifest.toml

```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.10.0"
+julia_version = "1.10.1"
```

Comment:
@juliasloan25 how do we handle this right now? If people on the team are using different julia versions. can they use the manifest made with Julia 1.10.0 even if running 1.10.1?

Reply:
Ah, I should have updated all the manifests to 1.10.1 when it came out but I didn't realize. I like to do it when I update the dependencies bc those changes update the manifest anyway. Maybe you could update the julia version as part of https://github.com/CliMA/ClimaLand.jl/pull/526 ? or I can open a separate PR for it

Reply:
I can take care of it! I'll probably do it in a separate PR